bin/graylogctl

```diff
@@ -51,7 +51,7 @@ GRAYLOG_CONF=${GRAYLOG_CONF:=/etc/graylog/server/server.conf}
 GRAYLOG_PID=${GRAYLOG_PID:=/tmp/graylog.pid}
 LOG_FILE=${LOG_FILE:=log/graylog-server.log}
 LOG4J=${LOG4J:=}
-DEFAULT_JAVA_OPTS="-Djdk.tls.acknowledgeCloseNotify=true -Xms1g -Xmx1g -XX:NewRatio=1 -server -XX:+ResizeTLAB -XX:+UseConcMarkSweepGC -XX:+CMSConcurrentMTEnabled -XX:+CMSClassUnloadingEnabled -XX:-OmitStackTraceInFastThrow"
+DEFAULT_JAVA_OPTS="-Djdk.tls.acknowledgeCloseNotify=true -Xms1g -Xmx1g -XX:NewRatio=1 -server -XX:+ResizeTLAB -XX:-OmitStackTraceInFastThrow"
```

Comment:
We cannot just remove the flags for all JVM versions. That would most probably the default GC for some JVMs. Please check the lines below on how to do a check for the presence of JVM flags so we can keep the flags for older JVMs.

Reply:
I was wondering about that. I didn't notice the code below it that checks if the flags are available. I'll see if I can add something like that.

Reply:
Okay, I added the changes you suggested. I tested them in an Ubuntu 20.04 VM using Java 8 and Java 15. On Java 8, it includes the CMS opts. On Java 15, it doesn't include them. Graylog starts successfully in both instances.